client/src/global-styles/GlobalStyles.tsx

```diff
@@ -7,11 +7,20 @@ const GlobalStyles = createGlobalStyle`
    margin: 0;
 }
 body {
-   font-family: 'Roboto', sans-serif;
+   font-family: 'Poppins', sans-serif;
    -webkit-font-smoothing: antialiased;
    -moz-osx-font-smoothing: grayscale;
    background-color: ${({ theme }) => theme.colors.common.backgroundcolor};
 }
 `;
+/*
+export const LogoStyle = createGlobalStyle`
```

Comment:
I couldn't add font for logo from exernal file

Reply:
1. Utwórz plik _fonts.d.ts_
2. declaration modules for fonts: 
`declare module '*.woff2';`
3. inside the _font_ folder, add a _fonts.ts_ file
4. add supported fonts e.g.
`import { createGlobalStyle } from "styled-components";
import plant_font from "./plants_lovin-webfont.woff2";

const FontStyles = createGlobalStyle`
@font-face {
    font-family: 'plants_lovin';
    src: url(${plant_font}) format('woff2');
    font-weight: normal;
    font-style: normal;
}
`;

export default FontStyles;`
5. In app.ts, add the component you export from fonts.ts under GlobalStyles

https://dev.to/alaskaa/how-to-import-a-web-font-into-your-react-app-with-styled-components-4-1dni